src/web/crate_details.rs

```diff
@@ -892,6 +900,50 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn test_crate_details_documentation_url_is_none_when_url_is_docs_rs() {
+        wrapper(|env| {
```

Comment:
this can be an async test, using `async_wrapper` (you'll find other tests in the codebase). 

you then also have to use `async_fake_release` and `create_async`. 

( the API need a better structure, but I'm finishing up some other things.

Reply:
@syphar thank you for the comment, I have changed the test to use async_wrapper :+1: